instagram/src/App.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import './App.css';
+import './components/PostContainer/Post.css';
+import './components/CommentSection/Comment.css';
+import dummyData from './dummy-data';
+import SearchHeader from './components/SearchBar/SearchHeader';
+import PostContainer from './components/PostContainer/PostContainer';
+
+class App extends Component {
+    state = {
+      posts: [],
+    filteredPosts: []
+    };
+
+componentDidMount() {
+  this.setState({
+    posts: dummyData,
+    filteredPosts: dummyData })
+}
+
+searchHandler = (searchterm) => {
+  let filtered = this.state.posts.filter(post => {
+    return post.includes(searchterm)
+  })
+  this.setState({
+    filteredPosts: filtered
+  })
+}
+
+render() {
+  return (
+    <div className="App">
+      <SearchHeader onChange={this.searchHandler} />
```

Comment:
putting it onChange is a good idea, but this just passes something called "onChange" to the SearchHeader